### ISSUE TITLE
Anti-cheating the Pope 67001

### ISSUE THREAD

**Issue by Aerotinge** [ck2plus/CleanSlate#136]
67001, the player is asked by the papal emissary to adopt papal investiture
https://github.com/ck2plus/CleanSlate/blob/2cba33cd451089c6b76273ceedc949d236769677/CleanSlate/events/base_pope_free_investiture.txt#L101-L106

However, the law is going to apply in option block, depending on the choice.
It gives the player a chance to swap their primary_title if there's more than one available, take the option and then swap back.
Leaving the original primary_title stays in free investiture.

I'd suggest a `primary_title = { save_event_target_as = title_being_demanded }` first in immediate block. So it always fires on the target title.

**Comment by Whizzer** [comment]
Good call, in the same vein as Paradox originally did to the Fabricate Claim-event. I'll keep this issue open in case any similar events ever come to mind. Feel free to change the issue's title, if you're able, or request me to do so if you're unable.
